godot-core/src/builtin/math/glam_helpers.rs

```diff
@@ -21,10 +21,12 @@ use crate::builtin::real;
 pub(crate) trait GlamConv {
     type Glam: GlamType<Mapped = Self>;
 
+    #[inline]
     fn to_glam(&self) -> Self::Glam {
         Self::Glam::from_front(self)
     }
 
+    #[inline]
```

Comment:
It's strange that this `#[inline]` seems to make a difference. To my knowledge, these annotations are only needed to enable _cross-crate_ inlining, and those are all private functions.

Reply:
Maybe I messed up while benchmarking, I'll test it again.